docs/roadmap.md

```diff
@@ -0,0 +1,49 @@
+# Roadmap to Federated Learning on Azure ML
+
+## Contents
+Here we propose a roadmap for onboarding to Federated Learning (FL) on Azure ML. The roadmap follows the Crawl-Walk-Run<!-- -Fly--> framework, where we suggest to start with a simple example, and then progressively add more complexity.
+
+## I. :turtle: **Crawl** - Kick the tires and get a feel for the technology
+
+Your first few steps should be to provision a basic setup for FL and try to run a simple example. This will help you get a feel for the technology and the Azure ML platform. Great news, our [quickstart](./quickstart.md) will help you do just that!
+
+>**:checkered_flag: :turtle: Checkpoint:** You are able to reproduce a simple FL training run.
+
+## II. :walking: **Walk** - Provision a custom setup, connect non-sensitive data, train first FL model
+
+Now that you are familiar with the basics of FL on Azure ML, we recommend you train a FL model to solve your business problem, working with non-sensitive data at first. This will help you understand the different components of the FL training process, and how to customize them. And should you make a mistake with the security settings or inadvertently expose some data, that is okay because it is non-sensitive data.
+
+First, you will want to provision a FL setup relevant to your use case (internal geographically-distributed silos, external silos, etc...). Start with our [provisioning README](./provisioning/README.md) to identify which ingredients you will need, and provision your custom setup using our Bicep templates. Note that since you will not be working with sensitive data, you don't have to use the templates using VNets` and Private Endpoints. If you do though, it will make the **Run** phase a bit easier, since your setup will already have these safety measures applied - up to you.
```

Comment:
I'd suggest create 2 tracks within this phase (under 2 sections), one for ramping up on the ML/DS side using synthetic data, the other on ramping up the infra requirements, with custom setup, etc.

Reply:
done